package.json

```diff
@@ -14,15 +14,17 @@
     "datastore"
   ],
   "dependencies": {
-    "async": "^2.6.1",
     "@azure/storage-blob": "^10.3.0",
+    "async": "^2.6.1",
     "azure-storage": "^2.10.2",
     "datastore-core": "~0.6.0",
     "fs": "0.0.1-security",
     "interface-datastore": "~0.6.0",
     "pull-defer": "~0.2.3",
     "pull-stream": "^3.6.9",
-    "upath": "^1.1.0"
+    "stream": "0.0.2",
+    "upath": "^1.1.0",
+    "util": "^0.11.1"
```

Comment:
Sanity: is this needed? From the `util` npm page, it says that there is no need for it for Node environment.